modules/cloudtrail/cloudtrail-cloudwatch-logs.tf

```diff
@@ -1,4 +1,6 @@
 data "aws_iam_policy_document" "cloudtrail_cloudwatch_logs_assume_role" {
```

Comment:
Where is this submodule used ? I don't see it used in main.tf

Reply:
here https://github.com/cloudposse/terraform-aws-components/pull/414/files#diff-1f16446afce3d91c5438027762e9141ea73e459243b443c3b9c994a1931d81a6R42